tkg/tkgconfigupdater/helper.go

```diff
@@ -212,6 +212,9 @@ func getFilesForChecksum(dirPath string) ([]string, error) {
 			return err
 		}
 
+		if info.IsDir() && (info.Name() == "yttcc" || info.Name() == "yttcb") {
```

Comment:
Can you explain the reason in commit message?

Reply:
@DanielXiao Add explanation into the commit message, pls check it, thanks!